libs/api/domains/document-provider/src/lib/dto/paperMail.input.ts

```diff
@@ -0,0 +1,13 @@
+import { Field, InputType, Int } from '@nestjs/graphql'
+import { IsNumber } from 'class-validator'
+
+@InputType()
+export class DocumentProviderPaperMailInput {
```

Comment:
~ Does the postholf api offer paginated responses? 

It is not following the API Design Guide on [Pagination](https://docs.devland.is/technical-overview/api-design-guide/pagination) 😅

Reply:
Yeah, same as with the document client. They only offer offset based pagination. 
Not sure if we can get them to change it since this is an old service.